bower_components/dust-demo/components/dust-demo-json.html

```diff
@@ -4,7 +4,7 @@
   #workspace {
     display: block;
     padding: 10px;
-    margin-bottom: 10px;
+    margin-bottom: 25px;
```

Comment:
These changes and any other changes in `bower_components` need to be made in their respective projects. This one can be found at https://github.com/smfoote/dust-demo